From b60fe68b9fee16b8b6dc91a5aa0019c3537bfecf Mon Sep 17 00:00:00 2001
From: Sam Sebastian <sebastiansam55@gmail.com>
Date: Mon, 29 Mar 2021 12:22:40 -0400
Subject: [PATCH] documentation patch

---
 lib/autokey/scripting/__init__.py | 12 ++++++------
 1 file changed, 6 insertions(+), 6 deletions(-)

diff --git a/lib/autokey/scripting/__init__.py b/lib/autokey/scripting/__init__.py
index af119d5..2520d51 100644
--- a/lib/autokey/scripting/__init__.py
+++ b/lib/autokey/scripting/__init__.py
@@ -33,9 +33,9 @@ from .system import System
 from .window import Window
 
 # Platform abstraction
-if autokey.common.USING_QT:
-    from .clipboard_qt import QtClipboard as Clipboard
-    from .dialog_qt import QtDialog as Dialog
-else:
-    from .clipboard_gtk import GtkClipboard as Clipboard
-    from .dialog_gtk import GtkDialog as Dialog
+# if autokey.common.USING_QT:
+from .clipboard_qt import QtClipboard
+from .dialog_qt import QtDialog
+# else:
+from .clipboard_gtk import GtkClipboard
+from .dialog_gtk import GtkDialog
-- 
2.25.1

